services/filemanager/filemanager.go

```diff
@@ -80,6 +84,13 @@ func GetProviderConfigFromEnv() map[string]interface{} {
 		providerConfig["accessKeyID"] = config.GetEnv("MINIO_ACCESS_KEY_ID", "minioadmin")
 		providerConfig["secretAccessKey"] = config.GetEnv("MINIO_SECRET_ACCESS_KEY", "minioadmin")
 		providerConfig["useSSL"] = config.GetEnvAsBool("MINIO_SSL", false)
+	case "DIGITAL_OCEAN_SPACES":
+		providerConfig["bucketName"] = config.GetEnv("JOBS_BACKUP_BUCKET", "")
+		providerConfig["prefix"] = config.GetEnv("JOBS_BACKUP_PREFIX", "")
+		providerConfig["endPoint"] = config.GetEnv("MINIO_ENDPOINT", "localhost:9000")
```

Comment:
Change these env variables to DO_SPACES_ENDPOINT and so on.

For any user who is configuring digital ocean for backups, it would be weird to configure MINIO env variables.